dynamodb_config_store/__init__.py

```diff
@@ -33,7 +33,12 @@
 """
 import os.path
 import time
-from ConfigParser import SafeConfigParser
+
```

Comment:
Please remove this blank line and group the imports from the standard library together.